docs/quickstart-and-usage.md

```diff
@@ -119,7 +127,13 @@ There are 2 additional flags for the `uds inspect` command you can use to extrac
 - Output the SBOMs as a tar file: `uds inspect ... --sbom`
 - Output SBOMs into a directory as files: `uds inspect ... --sbom --extract`
 
-This functionality will use the `sboms.tar` of the  underlying Zarf packages to create new a `bundle-sboms.tar` artifact containing all SBOMs from the Zarf packages in the bundle.
+This functionality will use the `sboms.tar` of the underlying Zarf packages to create new a `bundle-sboms.tar` artifact containing all SBOMs from the Zarf packages in the bundle.
+
+#### Viewing Variables
+
+To view the configurable overrides and Zarf variables of a bundle's packages:
+
+`uds inspect --list-variables <local-or-remote-tarball>:ref`
```

Comment:
Slight correction, when running `uds inspect --list-variables` on a local tarball, no need to include the ref

Reply:
same comment as above